### DISCUSSION TITLE
With Airbyte 0.57., /tmp/workspace volume owner is always resetted to root after recreation

### DISCUSSION THREAD

**Discussion by JonasDev1** [airbytehq/airbyte#37506] [Connector Questions]
Hey, 
I have upgraded Airbyte, now the Airbyte worker is running as user airbyte. 
Therefore I have recreated the workspace volume, this works fine. #36984 

If I reinstall my machine and only migrate the docker volumes folder(/var/lib/docker/volumes), this ownership setting is lost and owner is user root again. Afterwards I need to manually change it again. 

Is it possible to integrate the ownership setting in the dockerfile of the worker or does someone a workaround?

**Comment by SChalupaBraiins** [comment]
Manual ownership change described in #36984 works all the time, I used it to migrate our existing instance from 0.51.0 to 0.60.0. without re-creation of workspace.
Anyway, some auto-setup in dockerfile would be nice.

[a comment by an automated account is not shown]
